services/account/apps/api/src/services/accounts.service.ts

```diff
@@ -31,7 +31,7 @@ export class AccountsService {
           return { msaId, handle: handleResponse };
         }
         this.logger.log(`Failed to get handle for msaId: ${msaId}`);
-        return { msaId };
+        return { msaId: Number(msaId).toString() };
```

Comment:
Is this needed?
```suggestion
        return { msaId };
```

Reply:
For some reason, it was returning a hex value when I was testing it in swagger. Not happening in the other get account so kinda confused.

Reply:
If you were getting a hex value logged there, something else is going on... this log change might just be masking a more serious issue